make/agents/app/deploy.mk

```diff
@@ -0,0 +1,14 @@
+KUSTOMIZE_ARGS=--load-restrictor LoadRestrictionsNone
+KUSTOMIZE_OVERLAY=config/crd/overlays/agentapp
+
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
+.PHONY: install
+install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
+	$(KUSTOMIZE) build $(KUSTOMIZE_ARGS) $(KUSTOMIZE_OVERLAY) | kubectl apply -f -
```

Comment:
This doesn't do anything agent specific.  Are we intending on installing agent CRDs separately?

Reply:
agents will eventually install only a subset of the CRDs